test/connectors/CustomsFinancialsApiConnectorSpec.scala

```diff
@@ -330,6 +327,144 @@ class CustomsFinancialsApiConnectorSpec extends SpecBase {
     }
   }
 
+  "postCashAccountStatements" must {
+    "return success when calling the correct URL" in new Setup {
+
+      val expectedUrl = "apiEndpointUrl/accounts/cashaccountstatementrequest/v1"
+
+      when(requestBuilder.withBody(any())(any(), any(), any())).thenReturn(requestBuilder)
+
+      when(requestBuilder.execute(any[HttpReads[CashTransactions]], any[ExecutionContext]))
+        .thenReturn(Future.successful(accResponse))
+
+      when(mockHttpClient.post(any[URL]())(any())).thenReturn(requestBuilder)
+
+      val appWithMocks: Application = application
+        .overrides(
+          bind[HttpClientV2].toInstance(mockHttpClient),
+          bind[RequestBuilder].toInstance(requestBuilder)
+        ).build()
+
+      running(appWithMocks) {
+        val result = await(connector(appWithMocks).postCashAccountStatementRequest(
+          "eori", "can", fromDate, toDate))
+
+        result mustBe Right(accResponse)
+      }
+    }
+
+    "return failure when backend post fails" in new Setup {
+
+      private val responseCode: Int = 500
+
+      when(requestBuilder.withBody(any())(any(), any(), any())).thenReturn(requestBuilder)
+
+      when(requestBuilder.execute(any[HttpReads[Seq[CashDailyStatement]]], any[ExecutionContext]))
+        .thenReturn(Future.failed(new HttpException("It's broken", responseCode)))
+
+      when(mockHttpClient.post(any())(any())).thenReturn(requestBuilder)
+
+      val appWithMocks: Application = application
+        .overrides(bind[HttpClientV2].toInstance(mockHttpClient)).build()
+
+      running(appWithMocks) {
+        connector(appWithMocks).postCashAccountStatementRequest("eori", "can", fromDate, toDate).map {
+          _ mustBe Left(UnknownException)
+        }
+      }
+    }
+
+    "return error when passing error" in new Setup {
```

Comment:
Could you please update the description as below or on similar lines? as its for requestCouldNotBeProcessed

"return RequestCouldNotBeProcessed response when Request could not be processed error code is populated"